src/Tasks/Common/Resources/Strings.resx

```diff
@@ -342,7 +342,7 @@
     <comment>{StrBegin="NETSDK1056: "}</comment>
   </data>
   <data name="UsingPreviewSdkWarning" xml:space="preserve">
-    <value>NETSDK1057: You are working with a preview version of the .NET Core SDK. You can define the SDK version via a global.json file in the current project. More at https://go.microsoft.com/fwlink/?linkid=869452</value>
+    <value>NETSDK1057: You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview</value>
```

Comment:
@richlander The aka.ms link here redirects to https://docs.microsoft.com/en-us/dotnet/core/tools/global-json, which is just a generic global.json reference.  Is that the intent or will there be a different page more specific to .NET Core previews?

Reply:
cc @mairaw

Reply:
Microsoft's style guide asks for contractions (https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions):
You are -> You're

@dsplaisted I wasn't aware of that link going to our topic until a customer got confused with that page after getting this message. I was just planning to add a Troubleshooting area to that topic to explain some warnings you might get and what to do. There's also a warning from EF projects that are redirected to that same topic.

However, this simplification makes the warning even more cryptic in my opinion.

Reply:
Do we still want to take this change, then?

Reply:
I think we should either add back a sentence along the lines of "You can define the SDK version via a global.json file in the current project.", or create a more specific page for https://aka.ms/dotnet-core-preview to point to.

Reply:
That page is getting expanded https://github.com/dotnet/docs/pull/4704

Reply:
> That page is getting expanded dotnet/docs#4704

Given those updates to the target of the link, I think the message in this PR is OK.

Reply:
I'm fine with @dsplaisted 's suggested resolution here.